cloudinstall/placement/ui/__init__.py

```diff
@@ -165,9 +165,6 @@ def build_widgets(self):
         self.deploy_ok_msg = ("\u2713 All the required OpenStack services are "
                               "placed on a machine, and you can now deploy.")
 
-        # Save a new placements configuration once all required services are
-        # placed.
-        self.placement_controller.save()
```

Comment:
Should this be replaced with a call to do-autosave? Tbh I'm not totally clear on why we were saving here before either, so I don't know what the right answer is. 

In the very near future I will rework the placement code to record what is deployed and what still needs to be deployed, so maybe for that we will need to re save here.